wire-ios-data-model/Tests/Source/Model/ZMConnectionTests.m

```diff
@@ -159,32 +162,8 @@ - (void)testThatAcceptingAConnectionUnarchivesTheConversation;
     connection.status = ZMConnectionStatusAccepted;
 
     // then
-    XCTAssertNotNil(connection.conversation);
-    XCTAssertFalse(connection.conversation.isArchived);
-}
-
-- (void)testThatCancellingAConnectionDeletesTheUserRelationship;
```

Comment:
As mentioned in another comment, we no longer delete the relationship (and not sure why we did it in the first place).